### PR TITLE
Branch anna

### PR DESCRIPTION
**Pull request by annazelenka**
Added flavor page files and image files (we can delete them later once we have SQL set up). The code files have Dart Docs and I put the flavor page-specific files in their own folder (Flavor Page folder) to keep things clean.

### CODE REVIEW

File: lib/FlavorPage/flavor_page.dart [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      return await showDialog(context: context, builder: (_) => ReviewDialog(context: context));
+  }
+
+  @override
+  Widget build(BuildContext context) {
+        return Scaffold(
+        backgroundColor: Colors.grey[300],
+        appBar: AppBar(
+          title: Text('CS411 final project')
+          ,),
```
Comment (khalludi) [inline lib/FlavorPage/flavor_page.dart]: Minor nitpick, move comma for title into the previous line and make the close parentheses of AppBar in line with the `appBar` declaration.

File: lib/FlavorPage/review_card.dart [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+              fontSize: 18.0,
+              color: Colors.lightBlue,
+              )
+              ),
+              Text(
+              getFormattedDate(review.date),
+              style: TextStyle(
+              fontSize: 18.0,
+              color: Colors.black,
+              )
```
Comment (khalludi) [inline lib/FlavorPage/review_card.dart]: Fix indentation for better readability.

```suggestion
              Text(
                review.title,
                style: TextStyle(
                  fontSize: 18.0,
                  color: Colors.lightBlue,
                )
              ),
              Text(
                getFormattedDate(review.date),
                style: TextStyle(
                  fontSize: 18.0,
                  color: Colors.black,
                )
```

**Review by khalludi — COMMENTED** [review]
I love all the work and effort you've put into this. Overall, very well-written and organized code. I also like the comments that you've added.

There were a few indentations nitpicks, I only pointed out a few. Ideally, we'd have a code style checker that automatically reindented the code for us, so I don't expect you to go back and check everything manually.

The other issue is that you included all the images in the repository. I know that you need it for your tests to work, however, we also don't want to clog the repo with tons of photos. If you need all those photos to test the app, then I'm OK with it. Otherwise, try to cut down the photos to no more than 10.

After you make those tweaks, if Payal doesn't review by the end of the day, go ahead and merge 👍

**Comment by annazelenka** [comment]
Okay, thanks for the comments, Khalid! Unfortunately I don't have time today, but I'll fix everything up and merge by tomorrow night.

**Pull request merged**